core/controllers/question_editor_test.py

```diff
@@ -291,16 +403,47 @@ def test_get(self):
                 self.skill_id)
             self.logout()
 
-    def test_delete(self):
-        self.login(self.ADMIN_EMAIL)
+
+    def test_delete_with_new_structures_disabled_returns_404_status(self):
+        with self.swap(constants, 'ENABLE_NEW_STRUCTURE_EDITORS', False):
+            self.login(self.ADMIN_EMAIL)
+            self.delete_json(
+                '%s/%s' % (
+                    feconf.QUESTION_EDITOR_DATA_URL_PREFIX, self.question_id),
+                expected_status_int=404)
+            self.logout()
+
+    def test_delete_with_incorrect_question_id_returns_404_status(self):
+        with self.swap(constants, 'ENABLE_NEW_STRUCTURE_EDITORS', True):
+            self.login(self.ADMIN_EMAIL)
+            self.delete_json(
+                '%s/%s' % (
+                    feconf.QUESTION_EDITOR_DATA_URL_PREFIX, 'abc123456789'),
+                expected_status_int=404)
+            self.logout()
+
+    def test_delete_with_admin_email_deletes_question(self):
```

Comment:
For consistent naming style: `test_delete_with_admin_email_allows_question_deletion`.

Reply:
👍